src/core/xccl_schedule.c

```diff
@@ -0,0 +1,71 @@
+#include <xccl_schedule.h>
+#include <xccl_team_lib.h>
+#include <xccl_progress_queue.h>
+
+void ucc_event_manager_init(ucc_event_manager_t *em)
+{
+    int i;
+    for (i = 0; i < UCC_EVENT_LAST; i++) {
+        em->listeners_size[i] = 0;
+    }
+}
+
+void ucc_event_manager_subscribe(ucc_event_manager_t *em,
```

Comment:
@Sergei-Lebedev @lappazos all these interfaces should be xccl_ and not ucc_

Reply:
i'll change it in a different PR